cysync/src/pages/mainApp/sidebar/wallet/send/stepperForm.tsx

```diff
@@ -584,11 +585,17 @@ const SendForm: React.FC<StepperProps> = ({ stepsData, handleClose }) => {
   };
 
   const handleTransactionFeeChange = (e: any) => {
-    setTransactionFee(e.target.value);
+    setTransactionFee(
+      e.target.value
+        ? isBtcFork
+          ? Math.round(e.target.value)
+          : e.target.value
+        : e.target.value
+    );
```

Comment:
Dont do it like this
declare a new prop on CustomSlider that takes step and pass it here

Reply:
we need to think of a good solution that handles input and slider fields optimally